<a href="https://colab.research.google.com/github/a109010078/Computational-Thinking-and-Program-Design/blob/main/%E6%9F%AF%E4%B9%83%E7%91%9C_%E6%9C%9F%E6%9C%AB%E5%88%86%E7%B5%84%E4%BD%9C%E6%A5%AD.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

組員 A109010012 李芷涵、 A109010069 李賽瑋、A109010078 柯乃瑜、 A109010129 曾郁淇

In [ ]:
print("你好，初次見面～我是聊天機器人，讓我們開始聊天吧！")

name = input("嗨，請問妳叫什麼名字？: ")
print("你好啊", name, "很高興認識你><")

country = input("請問你從哪裡來? : ")
print("好棒:)聽說",country, "是個有趣的地方！")

print(name,"你是不是覺得計算圓的面積很麻煩呢?")
print("偷偷跟你說，我可以幫你計算喔！要不要考考我？")
print("輸入你要計算的圓半徑吧！")
r = int(input("r="))
def findArea(r):
    PI = 3.142
    return PI * (r*r)

print("圓面積為:", findArea(r))
print("這樣就算好了！你看，是不是很方便！")

print(name,"你想不想跟我玩一個遊戲呢?")
answer = input("想要跟我玩遊戲嗎(好啊/不要) :")


if answer == "好啊" :
     print("那我們要玩什麼遊戲?")
     print("不知道ㄟ 你有什麼想玩的嗎?")
     print("我想想喔!")
     print("那我們來玩猜數字遊戲好不好?")
     answer = input("要不要玩猜數字遊戲?(好啊/不要) :")
if answer == "好啊" :
    print("那我們來玩吧!")
    print("好!")
         
from random import choice
a = range(1,101)
t = choice(a)
game = True
while game:
    x = int(input("猜數字: "))
    if x > t:
        print("小一點")
    elif x < t:
        print("大一點")
    else:
        print(f"恭喜答對, 答案就是{t}, 遊戲結束")
        game = False

print("時間過得好快喔", name, "跟你聊天好開心～我們下次再見喔！")
    

你好，初次見面～我是聊天機器人，讓我們開始聊天吧！
嗨，請問妳叫什麼名字？: 1
你好啊 1 很高興認識你><
請問你從哪裡來? : 2
好棒:)聽說 2 是個有趣的地方！
1 你是不是覺得計算圓的面積很麻煩呢?
偷偷跟你說，我可以幫你計算喔！要不要考考我？
輸入你要計算的圓半徑吧！
r=3
圓面積為: 28.278
這樣就算好了！你看，是不是很方便！
1 你想不想跟我玩一個遊戲呢?
想要跟我玩遊戲嗎(好啊/不要) :好啊
那我們要玩什麼遊戲?
不知道ㄟ 你有什麼想玩的嗎?
我想想喔!
那我們來玩猜數字遊戲好不好?
要不要玩猜數字遊戲?(好啊/不要) :不要
猜數字: 1
大一點
猜數字: 6
大一點
猜數字: 11
大一點
猜數字: 100
小一點
猜數字: 49
小一點
猜數字: 23
大一點
猜數字: 34
大一點
猜數字: 55
小一點
猜數字: 45
小一點
猜數字: 44
小一點
猜數字: 43
小一點
猜數字: 42
恭喜答對, 答案就是42, 遊戲結束
時間過得好快喔 1 跟你聊天好開心～我們下次再見喔！


腳本：你好，初次見面～我是聊天機器人，讓我們開始聊天吧！
嗨，請問妳叫什麼名字？: 1
你好啊 1 很高興認識你><
請問你從哪裡來? : 2
好棒:)聽說 2 是個有趣的地方！
1 你是不是覺得計算圓的面積很麻煩呢?
偷偷跟你說，我可以幫你計算喔！要不要考考我？
輸入你要計算的圓半徑吧！
r=3
圓面積為: 28.278
這樣就算好了！你看，是不是很方便！
1 你想不想跟我玩一個遊戲呢?
想要跟我玩遊戲嗎(好啊/不要) :好啊
那我們要玩什麼遊戲?
不知道ㄟ 你有什麼想玩的嗎?
我想想喔!
那我們來玩猜數字遊戲好不好?
要不要玩猜數字遊戲?(好啊/不要) :不要
猜數字: 1
大一點
猜數字: 6
大一點
猜數字: 11
大一點
猜數字: 100
小一點
猜數字: 49
小一點
猜數字: 23
大一點
猜數字: 34
大一點
猜數字: 55
小一點
猜數字: 45
小一點
猜數字: 44
小一點
猜數字: 43
小一點
猜數字: 42
恭喜答對, 答案就是42, 遊戲結束
時間過得好快喔 1 跟你聊天好開心～我們下次再見喔！